charts/alfresco-activiti/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart for Alfresco Activiti
 name: alfresco-activiti
-version: 0.2.0-alpha.1
+version: 0.2.0-alpha.2
 appVersion: 24.3.0-SNAPSHOT
```

Comment:
isn't automated?

Reply:
It is but I reverted the change as the current code require unreleased 24.3.0 (and it's not used anywhere at that point)

Reply:
I will simply not release anything here for now

Reply:
ah ok, I was hoping the fix would be in the GA we are currently releasing